app/models/terms_and_conditions.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class TermsAndConditions < ActiveRecord::Base
+  has_many :terms_and_conditions_acceptances
+
+  scope :latest, -> { find_by(latest: true) }
+
+  validates :name, presence: true
```

Comment:
I thought we could see a case where there would be different terms of service for different domains, not limited to MHV terms. And those domains may require additional columns or methods on the model for handling outside the base case.